[AzureFW] Update azure FW log docs (#70564)
Azure/azure-content: articles/firewall/logs-and-metrics.md

   }
   ```

> [!NOTE]

> > When receiving the Deny response for HTTP requests under `default action` you will not find the logs under the Log Analytics Workspace with standard protocol or port queries.

Reviewer: ```suggestion
> When you receive a Deny response for HTTP requests under `default action`, you won't find the logs under the Log Analytics Workspace with standard protocol or port queries.
```

Author: Edited

> > The `Protocol` field will have a space in it and will match against `"HTTP "`, Or you can filter using the `msg_s` field's content.

Reviewer: ```suggestion
> The `Protocol` field will have a space in it and will match against `"HTTP "`, or you can filter using the `msg_s` field's content.
```

Author: Edited
